source/blog/2018-05-11-the-emberjs-times-issue-46.md

```diff
@@ -34,24 +52,10 @@ App.AboutRoute = Ember.Route.extend({
 });
 ```
 
-```
-// index.html
-<script type="text/x-handlebars" data-template-name="index">
-  <ul>
-    {{#each model as |item|}}
```

Comment:
I removed that code example because I believed the previous one already highlighted the use of the globals very well and since I felt anxious about the double curlies. 
We can't use double curlies in the text fields of our newsletter templates on goodbits (the e-mail just doesn't get sent out if we do so) and I think it's good to keep the content on the blog and in the newsletter as identical as possible (makes it also easier for me to move the content later on into goodbits)

cc @amyrlam @Gaurav0